src/data/events.ts

```diff
@@ -28,7 +28,7 @@ export const EVENTS: Event[] = [
         title: 'Meet and Greet',
         date: { year: 2024, month: 'MAR', day: 1, endTime: '21:00' },
         time: '5:00pm - 9:00pm',
-        location: 'Engineering and Maths EM105',
+        location: 'Engineering and Maths 105',
```

Comment:
The room is called EM105 so it should stay that way

Reply:
Who told you to change it?

Reply:
@phoenixpereira

Reply:
Ah okay, yeah it's mentioned in #154. That was an error - "EM" should be included in the name.

Sources:
- https://www.adelaide.edu.au/student/exams/ua/media/18/directions-to-EM205-EM213-EM218.pdf
- https://studentvip.com.au/adelaide-uni/north-terrace/maps/127015

Reply:
Ah okay, change it back to EM105 then.